build.psm1

```diff
@@ -89,6 +89,95 @@ function Test-Win10SDK {
     return (Test-Path "${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64\mc.exe")
 }
 
+function Start-BuildNativeWindowsBinaries {
+    param(
+        [ValidateSet('Debug', 'Release')]
+        [string]$Configuration = 'Release',
+
+        [ValidateSet('x64', 'x86')]
+        [string]$Arch = 'x64'
+    )
+
+    if (-not $Environment.IsWindows) { return }
+
+    # cmake is needed to build powershell.exe
+    if (-not (precheck 'cmake' $null)) {
+        throw 'cmake not found. Run "Start-PSBootstrap -Package". You can also install it from https://chocolatey.org/packages/cmake'
```

Comment:
Why not introduce a new `-BuildNativeWindows` switch instead of reusing `-Package`?  That way I can privately build a package w/o needing the SDK or building pwshplugin locally.